homeassistant/components/lifx/light.py

```diff
@@ -52,6 +59,15 @@
     }
 )
 
+SERVICE_LIFX_SET_HEV_CYCLE_STATE = "set_hev_cycle_state"
+
+LIFX_SET_HEV_CYCLE_STATE_SCHEMA = cv.make_entity_service_schema(
+    {
+        ATTR_POWER: vol.Required(cv.boolean),
+        ATTR_DURATION: vol.All(vol.Coerce(float), vol.Clamp(min=0, max=86400)),
+    }
+)
```

Comment:
```suggestion
LIFX_SET_HEV_CYCLE_STATE_SCHEMA = {
     ATTR_POWER: vol.Required(cv.boolean),
     ATTR_DURATION: vol.All(vol.Coerce(float), vol.Clamp(min=0, max=86400)),
}
```

The `async_register_entity_service` should wrap this for you